pytorch_lightning/plugins/ddp_plugin.py

```diff
@@ -108,3 +108,25 @@ def on_before_forward(self, model, *args):
 
     def optimizer_state(self, optimizer: Optimizer) -> dict:
         return optimizer.state_dict()
+
+    def module_from_plugin(
```

Comment:
I find the name confusing here: module_from_plugin

Reply:
would `model_from_plugin` be better? The issue is it's usually a module stored within the wrapper class, so ties to all parallel implementations already